lib/sass/calcite-web/patterns/_sub-nav.scss

```diff
@@ -9,25 +9,31 @@
   height: 70px;
 }
 
-@mixin sub-nav-title() {
+@mixin sub-nav-container() {
+  display: flex;
+  align-items: center;
+}
+
+@mixin sub-nav-title() { 
   @include font-size(2);
-  margin: $baseline*.8 0 0 $baseline;
-  float: left;
-  display: inline;
+  margin: 0 0 0 1rem;
   text-overflow: ellipsis;
   overflow: hidden;
   white-space: nowrap;
   min-width: 0;
+  display: flex;
+  justify-content: flex-start;
 }
 
 @mixin sub-nav-list() {
   width: auto;
-  @include right();
+  display: flex;
+  justify-content: flex-end;
+  flex: 1;
```

Comment:
Did you test this in different browsers? I think IE11 and some other browsers have different default implicit values for the single unit shorthand of the flex property.

Reply:
Tested in IE11, Edge, Firefox, Safari, and Chrome. No change.